Server/V2/controller/userController.js

```diff
@@ -1,10 +1,44 @@
 import UserModel from '../model/userModel';
-import { serverError, serverResponse } from '../Helper/Response';
+import sql from '../db/sqlQueries';
+import dbQuery from '../db/config';
+import { generateToken } from '../Middleware/Auth';
+import {serverResponse } from '../Helper/Response';
 import { encryptPassword, decryptPassword } from '../Helper/encrypt';
-import {SignupValidator, loginValidator} from '../Middleware/Validations'
-
 
 const User = {
+    async create(req, res) {
+        try{
+        const {firstname, lastname, email, phoneNumber, username, password, isAdmin} = req.body;
+        const allusers = await dbQuery(sql.allUsers);
+        const id = parseInt(allusers.forEach) + 1;
```

Comment:
@KarangwaEmmy Instead of manually creating the id, you can retrieve the id of the newly created account